src/rust/src/sapling.rs

```diff
@@ -343,21 +366,51 @@ impl BatchValidator {
     /// `sighash` must be for the transaction this bundle is within.
```

Comment:
```suggestion
    /// `sighash` must be for the transaction this bundle is within.
    ///
    /// If this batch was configured to not cache the results, then if the bundle was in
    /// the global bundle validity cache, it will have been removed (and this method will
    /// return `true`).
```